client/pwa/src/service-worker.ts

```diff
@@ -94,7 +91,7 @@ self.addEventListener("message", (e: ExtendableMessageEvent) => {
 
         case "keepalive":
           console.log("[keepalive]");
-          return unpacking;
+          return;
```

Comment:
@caugner: This was returning the `unpacking` for the `e.waitUntil` so it keeps the service worker alive. Unless something changed we must keep this.

Reply:
Ouch, good catch, I must have mixed things up here. Fixed now!